core/src/main/kotlin/org/jetbrains/kotlinx/dataframe/aggregation/AggregateDsl.kt

```diff
@@ -11,6 +12,7 @@ import org.jetbrains.kotlinx.dataframe.impl.columnName
 import kotlin.reflect.KProperty
 import kotlin.reflect.typeOf
 
+@HasSchema(schemaArg = 0)
```

Comment:
What does HasSchema do again? There's no docs :/

Reply:
To help identify type parameter that represents dataframe schema
org.jetbrains.kotlinx.dataframe.plugin.InterpretKt#getSchema
org/jetbrains/kotlinx/dataframe/plugin/interpret.kt:255

Reply:
right, so if we would take the type parameter at `schemaArg`, which in `AggregateDsl<out T>` is `out T`, and put it in `DataFrame<>` like `DataFrame<T>`, a schema generated for thát would represent this object. And this could be different for types like `GroupedDataRow<out T, out G>` where `G` might represent the data schema?

Could you explain this in a small piece of KDoc?

Reply:
I'll think of something